ext/openssl/ossl_pkey.c

```diff
@@ -1785,6 +1802,9 @@ Init_ossl_pkey(void)
     rb_define_method(cPKey, "encrypt", ossl_pkey_encrypt, -1);
     rb_define_method(cPKey, "decrypt", ossl_pkey_decrypt, -1);
 
+
+    rb_define_method(cPKey, "keysize_in_bits", ossl_pkey_length_in_bits, 0);
```

Comment:
This line should be moved to right below `inspect`, and ossl_pkey_length_in_bits() definition to below ossl_pkey_inspect().